src/Bundle/CoverallsBundle/Collector/CiEnvVarsCollector.php

```diff
@@ -64,7 +64,8 @@ public function collect(array $env)
             ->fillAppVeyor()
             ->fillJenkins()
             ->fillLocal()
-            ->fillRepoToken();
+            ->fillRepoToken()
+            ->fillGithubActions();
```

Comment:
To me it would make sense to have this below Jenkins. To keep CI services related methods together.

And to modify the methods order accordingly in this class.

Reply:
I move call method after `fillJenkins()`